include/multigrid.h

```diff
@@ -394,9 +394,8 @@ namespace quda {
 
     /**
        @brief Load the null space vectors in from file
-       @param B Loaded null-space vectors (pre-allocated)
     */
-    void loadVectors(std::vector<ColorSpinorField *> &B);
+    void loadVectors();
```

Comment:
Just wondering what the rationale was for removing the argument to this function, but retaining it for `saveVectors` and `initializeRandomVectors`?

Reply:
No idea! I'll add it back.

Reply:
👍